code/datums/recipe.dm

```diff
@@ -54,14 +54,26 @@ var/global/list/microwave_accepts_items =    list(
 	if(length(container_contents) < length(fruit))
 		return FALSE
 	var/list/needed_fruits = fruit.Copy()
-	for(var/obj/item/chems/food/snacks/grown/G in container_contents)
-		var/ktag = G.seed?.kitchen_tag
-		if(needed_fruits[ktag] > 0)
-			needed_fruits[ktag]--
+	for(var/obj/item/chems/food/snacks/S in container_contents)
+		var/use_tag
+		if(istype(S, /obj/item/chems/food/snacks/grown))
+			var/obj/item/chems/food/snacks/grown/G = S
+			if(!G.seed || !G.seed.kitchen_tag)
```

Comment:
Something for future consideration would be having an extension to hold seed information rather than duplicating vars between /grown, /fruit_slice and hydro trays.